mailpoet/lib/Automation/Engine/Migrations/Migrator.php

```diff
@@ -60,12 +60,28 @@ public function createSchema(): void {
         PRIMARY KEY (id)
       );
     ");
+
+    $this->runQuery("
+      CREATE TABLE {$this->prefix}workflow_run_logs (
+        id int(11) unsigned NOT NULL AUTO_INCREMENT,
+        workflow_run_id int(11) unsigned NOT NULL,
+        step_id varchar(255) NOT NULL,
+        status varchar(255) NOT NULL,
+        started_at timestamp NOT NULL,
```

Comment:
Do you think it would be useful to have `created_at` in addition to `started_at`? I think they would be the same every time, unless we wanted `created_at` to correspond to when the record is actually saved, in which case I think it would always be the same as `completed_at`.

Reply:
Yeah, I think I don't see a need for `created_at` at the moment.